app/templates/test_fill_in_the_blank.html

```diff
@@ -4,12 +4,66 @@
 {% block instructions %}
     <div class="row">
         <div class="col">
-            <strong>Instructions:</strong> Fill in the blanks with the correct answer(s).
-            All textboxes must be filled.
+            <strong>Instructions:</strong> Fill in the blank(s) with the correct answer(s).
         </div>
     </div>
 {% endblock %}
 
 {% block response_area %}
-    <h2>Fill in the blank response area</h2>
-{% endblock %}
\ No newline at end of file
+<div class="row pt-2" id="response_area">
+    <form method="POST"
+          action="{% if not preview_mode %}{{ post_url }}{% endif %}">
+        {{ form.csrf_token }}
+        {{ form.question_id }}
+
+        {% set class_str = "form-control " + forms.get_valid_class(form.response) %}
+        {{ form.response }}
+        {% if form.response.errors %}
+            <div class="ms-1 invalid-feedback">
+                Enter an answer or click "I Don't Know".
+            </div>
+        {% endif %}
+        <p class="pt-2">
+            {{ form.no_answer(class_="btn btn-secondary btn-sm") }}
+            {{ form.submit(class_="btn btn-primary btn-sm") }}
+        </p>
+<script>
+
+    let blank_box_num = 1;
+    while (document.getElementById(`FITB${blank_box_num}`) != null){ //adding event listeners for every textbox
+        document.getElementById(`FITB${blank_box_num}`).addEventListener("change", update_user_response)
+        blank_box_num++;
+        }
+
+    function update_user_response() {
```

Comment:
This seems to be duplicated from the fill-in-the-blank.js file. The function should live there. You will likely need to include that file to make it accessible.